src/main/java/uk/gov/companieshouse/missingimagedelivery/orders/api/util/StringHelper.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.companieshouse.missingimagedelivery.orders.api.util;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Component;
+
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Component
+public final class StringHelper {
+
+    public Set<String> asSet(String regexDelim, String values) {
+        return Stream.of(Optional.ofNullable(values).orElse("").split(regexDelim))
+                .filter(StringUtils::isNotEmpty)
+                .collect(Collectors.toSet());
```

Comment:
Could this be a static method or class? Also this is just a personal opinion but I've always considered streams to be not the best when it comes to readability, debugging and optimisation.

Reply:
As I'm mocking it in my tests, I won't be able to make use static here so using the Spring @ Component.  
About the streams,  think for the purpose of this small utils method where it's the only thing in the class streams can be handy, cuts down some loads of lines of code?

Reply:
I'm of the opinion that utility methods aren't really what mocking is for, but if you're happy with the design I'm happy to leave it.

Reply:
Fair -  I'll need the component still for autowired part  (so non static) to work but happy to refactor the Eric Authoriser where it is being mocked  [here](https://github.com/companieshouse/missing-image-delivery.orders.api.ch.gov.uk/blob/f93412542b9d77faf68ca2a1b2767abdcf66fe29/src/test/java/uk/gov/companieshouse/missingimagedelivery/orders/api/interceptor/EricAuthoriserTest.java#L28):